sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1361,6 +1363,12 @@ export interface OperationBase {
 // @public (undocumented)
 export type OperationInput = CreateOperationInput | UpsertOperationInput | ReadOperationInput | DeleteOperationInput | ReplaceOperationInput | PatchOperationInput;
 
+// @public (undocumented)
+export interface OperationOptions {
```

Comment:
I would suggest renaming this to `QueryOperationOptions`. Here is my rationale. We want to distinguish between `FeedOptions` which applies on higher level (iterator creation) but retain a hint that it applies to Query operations. Also there doesn't seem possibility of reusing this for other purposes.